### ISSUE TITLE
Remove React dependency

### ISSUE THREAD

**Issue by JoviDeCroock** [FormidableLabs/urql-devtools-exchange#25]
We are looking at React internals to derive the `displayName` we could defer this to the integration specific implementation and make this a dev-only thing.

**Comment by andyrichardson** [comment]
Brownie points if we can somehow infer the calling components displayName in Preact also

**Comment by JoviDeCroock** [comment]
```
import { options } from 'preact';

v?: Vnode;
if (!isProd) options.vnode = (v) => { currentVNode = v }

const hook = () => dispatch({ context: { currentComponent: v&&v.name })
```

**Comment by andyrichardson** [comment]
Fixed in #28
